### PR TITLE
[RHACS] ROX-24496: Improving content in the Managing Compliance 2.0 section

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.4+

Issue:
https://issues.redhat.com/browse/ROX-24496

Link to docs preview:
- https://76873--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/manage-compliance/managing-compliance-20.html
- https://76873--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/manage-compliance-operator/compliance-operator-rhacs.html

SME review:
- [ ] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.5`
    - `rhacs-docs-4.4`

### CODE REVIEW

**Review by dashrews78 — APPROVED** [review]
LGTM

File: modules/compliance-operator-configure-scanning.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to
 ====
 * If you are using the compliance 2.0 feature, you can schedule your scan by using {product-title-short} to create a compliance scan schedule instead of creating the `ScanSettingBinding` on the Compliance Operator.
 +
-For more information about scheduling a compliance scan by using the compliance 2.0 feature, see "Creating a compliance scan schedule" in the "Additional resources” section.
+For more information about scheduling a compliance scan by using the compliance 2.0 feature, see "Customizing and automating your compliance scans" in the "Additional resources” section.
```
Comment (kcarmichael08) [inline modules/compliance-operator-configure-scanning.adoc:16]: For future PRs, the guidance now is to not use the "in the Additional resources section" part. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

**Review by kcarmichael08 — COMMENTED** [review]
one thing for future PRs, but I think it's OK to leave it for this one. Everything else lgtm.

File: modules/compliance-operator-configure-scanning.adoc (hunk 1 above)
Comment (agantony) [reply modules/compliance-operator-configure-scanning.adoc:16]: Thanks for calling this out. I will create a separate task to address this across ACS documentation.

**Pull request merged**